### PR TITLE
fix(fx-2400): tabs spacing fixes

### PR DESCRIPTION
**Pull request by dzucconi**
Closes: [FX-2400](https://artsyproduct.atlassian.net/browse/FX-2400)

### Before:
 www artsy net_artist_katie-paterson(Capture)

### After:
localhost_5000_search_term=painting(Capture)

### CODE REVIEW

File: src/v2/Components/Pagination.tsx [hunk 1]
```diff
@@ -35,11 +35,15 @@ export class Pagination extends React.Component<Props> {
...
         <Media at="xs">
+          {/* FIXME: */}
+          {/* @ts-ignore */}
           <SmallPagination {...this.props} />
         </Media>
         <Media greaterThan="xs">
           <Box>
             <Separator mb={3} pr={2} />
+            {/* FIXME: */}
+            {/* @ts-ignore */}
```
Comment (dzucconi) [inline src/v2/Components/Pagination.tsx:46]: Ah, I see @eessex beat me to this @jonallured — this should be fixable. I'll still maintain we don't need to pass in the cursors, but there are other ways to fix this too

**Review by dblandin — APPROVED** [review]
Looks great! 💯

**Comment by dblandin** [comment]
Looks like a spec needs an update for that space -> &nbsp; change.

**Comment by dblandin** [comment]
Assigned back to you to merge when ready 👍

**Pull request merged**